src/master/heartbeat_cluster.cpp

```diff
@@ -471,6 +464,176 @@ namespace cubhb
     return valid_ping_host_exists;
   }
 
+  void
+  cluster::handle_heartbeat (const heartbeat_arg &arg, ipv4_type from_ip)
+  {
+    if (shutdown || m_hostname != arg.get_dest_hostname ())
+      {
+	// if the cluster is stopped or there is a mismatch on hostname then exit
```

Comment:
For `m_hostname != arg.get_dest_hostname ()`, it used to call MASTER_ER_LOG_DEBUG. Is log removed intentionally?

Reply:
I added log message back